package.json

```diff
@@ -47,16 +49,25 @@
     "@babel/core": "^7.22.1",
     "@babel/preset-env": "^7.22.4",
     "@babel/preset-typescript": "^7.21.5",
+    "@jest/globals": "^29.5.0",
+    "@nextcloud/eslint-config": "^8.3.0-beta.2",
     "@nextcloud/event-bus": "^3.1.0",
     "@rollup/plugin-typescript": "^11.1.1",
+    "@types/jasmine": "^4.3.4",
     "babel-jest": "^29.5.0",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
     "rollup": "^3.23.0",
+    "ts-node": "^10.9.1",
     "typescript": "^5.0.4"
   },
   "engines": {
     "node": "^20.0.0",
     "npm": "^9.0.0"
+  },
+  "eslintConfig": {
+    "extends": [
+      "@nextcloud/eslint-config/typescript"
+    ]
```

Comment:
I prefer a dedicated file like all other repos :)

Reply:
One file for 4 lines of code :stuck_out_tongue_winking_eye:

Reply:
Sure, but then we all know where to look :)
It's about standardizing